AGGREGATE.md

```diff
@@ -69,25 +69,30 @@ example an ad-tech will use the API to collect:
 * Aggregate purchase values at a per geo level 
 ### Attribution source registration
 
-Registering sources eligible for aggregate reporting entails adding a new header
-on the response to the `attributionsrc` request:
-`Attribution-Reporting-Register-Aggregatable-Source`, in the form of a JSON
-list.
+Registering sources eligible for aggregate reporting entails adding a new
+`aggregation_keys` list field to the JSON dictionary of the
+[`Attribution-Reporting-Register-Source` header](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#registering-attribution-sources):
 ```jsonc
-[{
-  // Generates a "0x159" key piece (low order bits of the key) for the key named
-  // "campaignCounts"
-  "id": "campaignCounts",
-  "key_piece": "0x159" // User saw ad from campaign 345 (out of 511)
-},
 {
-  // Generates a "0x5" key piece (low order bits of the key) for the key named "geoValue"
-  "id": "geoValue",
-  // Source-side geo region = 5 (US), out of a possible ~100 regions.
-  "key_piece": "0x5"
-}]
+  ... // existing fields, such as `source_event_id` and `destination`
+
+  "aggregation_keys": [
```

Comment:
Should this be aggregatable_keys to match the rest of the naming? Ditto for the prose